quartus/cave.qsf

```diff
@@ -24,7 +24,7 @@ set_global_assignment -name MIN_CORE_JUNCTION_TEMP 0
 set_global_assignment -name MAX_CORE_JUNCTION_TEMP 85
 set_global_assignment -name POWER_PRESET_COOLING_SOLUTION "23 MM HEAT SINK WITH 200 LFPM AIRFLOW"
 set_global_assignment -name POWER_BOARD_THERMAL_MODEL "NONE (CONSERVATIVE)"
-set_global_assignment -name TIMING_ANALYZER_MULTICORNER_ANALYSIS OFF
+#set_global_assignment -name TIMING_ANALYZER_MULTICORNER_ANALYSIS OFF
```

Comment:
You can also just change this line to `set_global_assignment -name TIMEQUEST_MULTICORNER_ANALYSIS OFF` instead of commenting it out. The name of this changed at some point after 17.0.